### PR TITLE
Prepare nf-core tools release 1.6

### PR DESCRIPTION
**Pull request by sven1103**
A bunch of changes to introduce for a new nf-core tools release.

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ If you're not used to this workflow with git, you can start with some [basic doc
 
 ## Style guide
 Google provides an excellent [style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md), which
-is an best practise extension of [PEP](https://www.python.org/dev/peps/), the Python Enhancement Proposals. Have a look at the 
+is an best practise extension of [PEP](https://www.python.org/dev/peps/), the Python Enhancement Proposals. Have a look at the
```
Comment (apeltzer) [inline .github/CONTRIBUTING.md]:
```suggestion
is a best practise extension of [PEP](https://www.python.org/dev/peps/), the Python Enhancement Proposals. Have a look at the
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
 # nf-core/tools: Changelog
 
+## v1.6dev
```
Comment (apeltzer) [inline CHANGELOG.md]:
```suggestion
## v1.6
```

Comment (apeltzer) [reply CHANGELOG.md]: Also we need a _new  fancy name_ !!!

File: setup.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 import sys
 
-version = '1.5'
+version = '1.6dev'
```
Comment (apeltzer) [inline setup.py]:
```suggestion
version = '1.6'
```

File: README.md [hunk 4]
```diff
@@ -85,16 +100,109 @@ nf-core list rna rna-seq
...
+```
 
 Finally, to return machine-readable JSON output, use the `--json` flag.
 
+## Launch a pipeline
+Some nextflow pipelines have a considerable number of command line flags that can be used.
+To help with this, the `nf-core launch` command uses an interactive command-line wizard tool to prompt you for
+values for running nextflow and the pipeline parameters.
+
+If the pipeline in question has a `parameters.settings.json` schema, parameters will be grouped and have associated description text and variable typing.
```
Comment (sven1103) [inline README.md]:
```suggestion
If the pipeline in question has a `parameters.settings.json` file following the [nf-core parameter JSON schema](https://nf-co.re/parameters.schema.json), parameters will be grouped and have associated description text and variable typing.
```

**Comment by ewels** [comment]
This pull request **introduces 1 alert** and **fixes 1** when merging 95624ed47077d4cbb43512ba85514dd601d9449b into 1855d346a8f1c4b839ab7d2264212edc478e1912 - [view on LGTM.com](https://lgtm.com/projects/g/nf-core/tools/rev/pr-5807990510d651419db92fcf9668ef608210d237)

**new alerts:**

* 1 for Except block handles &#39;BaseException&#39;

**fixed alerts:**

* 1 for Unused local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**